examples/gno.land/p/gnoland/boards/reply.gno

```diff
@@ -0,0 +1,55 @@
+package boards
+
+import (
+	"errors"
+	"strings"
+	"time"
+
+	"gno.land/p/nt/ufmt"
+)
+
+// NewReply creates a new reply to a thread or another reply.
+func NewReply(parent *Post, creator address, body string) (*Post, error) {
```

Comment:
Something to consider here could be to maybe define `NewReply()`, `NewThread()` and `NewRepost()` in the Boards2 realm, because they have logic which could be considered realm related if you think of boards package as a generic minimal package.

It might be helpful to hear second opinions.